decidim-assemblies/app/models/decidim/assembly_member.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Decidim
+  # It represents a member of the assembly (president, secretary, ...)
+  # Can be linked to an existent user in the platform
+  class AssemblyMember < ApplicationRecord
+    include Decidim::Traceable
+    include Decidim::Loggable
+
+    GENDERS = %w(man woman).freeze
```

Comment:
@decidim/product do we really need to save the gender? I'm not an expert but in terms of LOPD isn't this considered a different level of data?

Also, if we really want to know the gender of the members, maybe we should consider an extra field with a free text option.

Reply:
Hi @oriolgual, this is a specific requirement from Barcelona Participation Regulation, which states that the composition of assemblies must be proportional in terms of age, gender and origin.

That said, I think a drop-down field with binary option is not a good solution. As the gender issue can be quite tricky (see the 71 gender options offered by Facebook) I would leave this field as an open text field.

In the end, it should serve to make parity issues visible. I understand that using an open text field is the simplest change.